src/components/menu-right/DotTypeEditor.vue

```diff
@@ -1,6 +1,18 @@
 <template>
   <div class="my-2">
-    <p data-test="menu-right--dot-type">Dot Type {{ dotTypeIndex }}</p>
+    <b data-test="menu-right--dot-type">Dot Type {{ dotTypeIndex }}</b>
```

Comment:
Nit: We still want this to be a `p` element to be semantically correct here. It's a small detail, so no big deal, but in general I like to properly format the HTML if possible! Typically you see `b` being used to style text within a `p` block.

```suggestion
    <p data-test="menu-right--dot-type"><b>Dot Type {{ dotTypeIndex }}</b></p>
```

Alternatively create a CSS class with `font-weight: bold` and whatever other styling changes you need! (Probably `display: inline;`?